master/buildbot/util/git.py

```diff
@@ -165,7 +167,9 @@ def _getSshDataPath(self):
             parent_path = path_module.join(self.worker.worker_basedir,
                                            path_module.dirname(workdir))
 
-        basename = '.{0}.{1}.buildbot'.format(self.build.builder.name,
+        safe_builder_name = safeTranslate(self.build.builder.name)
+        safe_builder_name = bytes2unicode(safe_builder_name)
```

Comment:
Maybe put the above two function calls into a single line?

Reply:
I think I need to rework this anyway, to use `self.build.builder.config.workerbuilddir`